src/gui/LayoutMapper.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Xournal++
+ *
+ * A layout manager - map where( row,column) to which page( document index)
+ *
+ * @author Justin Jones
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+#include <XournalType.h>
+
+enum LayoutType {					//		1		2		4
+	Horizontal = 0,						//									
+	Vertical = 1,							//		V							
+	Horizontal_RL = 2,				//				RL			
+	Vertical_RL = 3,						//		V		RL		
+	Horizontal_BT = 4,				//						BT		
+	Vertical_BT = 5,						//		V				BT	
+	Horizontal_RL_BT = 6,			//				RL		BT			
+	Vertical_BT_RL = 7,				//		V		RL		BT		
+	BitFlagsUsedToHere = 8,		//do not modify this or above
+
+};
+
+enum LayoutBitFlags{
+		ColMajor = 1,		//aka Vertical
```

Comment:
When you are using Vertical everywhere else just name it here also vertical.

Reply:
It's a bit more complicated as I learned here: https://stackoverflow.com/questions/8357240
I changed it to 'Column' as that's a bit cleaner.